docs/release-notes.rst

```diff
@@ -4,11 +4,25 @@
 Release Notes
 =============
 
-v2023.MM.00 (MM DD, YYYY)
+v2023.05.0 (May 4, 2023)
 -------------------------
 
-Bug Fixes
-^^^^^^^^^
+Management of this package switched to GeoCAT team members `Julia Kent`_` and `Katelyn Fitzgerald`_`.
+
+New Features
+^^^^^^^^^^^^
+* Pressure-height twin axis function `add_height_from_pressure_axis` in (:pr:`114`)
 
 Documentation
 ^^^^^^^^^^^^^
+* Contributor's Guide updated in (:pr:`115`)
+* Documentation and Packaging Overhaul to match GeoCat-Comp style in (:pr:`116`), (:pr:`121`)
+* Adding examples links to all functions in (:pr:`122`)
+
+**Full Changelog**: https://github.com/NCAR/geocat-viz/compare/v2023.03.0...v2023.05.0
```

Comment:
@pilotchute Is this the right thing to do for the changelog link? That link doesn't exist until the release.

Reply:
@jukent Yes, that's correct. We're actually omitting this in geocat-comp now, but this was the way we did it up until this coming release.

Reply:
Okay I'll follow suit and remove it here too. The broken link seems annoying every time.